scipy/stats/morestats.py

```diff
@@ -2721,50 +2722,109 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False):
         The second set of measurements.  If `y` is not given, then the `x`
         array is considered to be the differences between the two sets of
         measurements.
-    zero_method : string, {"pratt", "wilcox", "zsplit"}, optional
+    zero_method : {"pratt", "wilcox", "zsplit"}, optional
         "pratt":
-            Pratt treatment: includes zero-differences in the ranking process
-            (more conservative)
+            includes zero-differences in the ranking process,
+            but drop ranks of the zeros, see [4]_, (more conservative)
```

Comment:
maybe "drops rank" or something else? I didn't understand this sentence.

Reply:
yes, "drops ranks"